tests/ui_build/option_not_sized.stderr

```diff
@@ -0,0 +1,13 @@
+error[E0080]: evaluation of `cxx::RustOption::<T>::new::__SizeCheck::<&dyn std::fmt::Debug>::_IS_USIZE_SIZE` failed
+ --> src/rust_option.rs
+  |
+  |                 assert!(core::mem::size_of::<Option<U>>() == core::mem::size_of::<usize>());
+  |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ the evaluated program panicked at 'assertion failed: core::mem::size_of::<Option<U>>() == core::mem::size_of::<usize>()', $DIR/src/rust_option.rs:38:17
```

Comment:
This one is annoying because the error message depends on the line of code in rust_option.rs. Ideally we could ignore the line number? Not sure if that is possible